test/Microsoft.AspNetCore.SignalR.Tests/WebSocketsTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SignalR.Tests
+{
+    [CollectionDefinition(Name)]
+    public class FunctionalTestsCollection : ICollectionFixture<ServerFixture>
```

Comment:
Rename to EndToEndTestsCollection and move to a separate file. (FunctionalTests is confusing since there is a separate called FunctionalTests). 

Side note: we need to clean the mess with the test projects we have many projects but most of them don't have many tests (or at all)